pkg/plugins/golang/v3/scaffolds/internal/templates/api/conversation.go

```diff
@@ -0,0 +1,101 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
+)
+
+var _ machinery.Template = &Conversion{}
+
+// Conversion scaffolds the methods of Hub interface for conversion webhook
+type Conversion struct {
+	machinery.TemplateMixin
+	machinery.BoilerplateMixin
+	machinery.ResourceMixin
+
+	// Version refers to the conversion webhook versions for hub and spoke.
+	Version string
```

Comment:
Could you please elaborate on how the command would like like? I remember there were many questions and discussions on the previous PR regarding the same. It will really helpful to revisit the command UX.